src/Node/Render.php

```diff
@@ -9,13 +13,31 @@
 use Twig_Node_Expression_Constant;
 use Twig_Node_Include;
 
+/**
+ * Class to create and compile a new 'Render' node Twig template.
+ *
+ * Components are rendered within their own context. The default variables are
+ * set from the component config file if not given explicitly via the tag.
+ * Variants must be referenced with a double hyphen and be configured in the main
+ * component config. – Having a separate variant file is currently not supported.
+ */
```

Comment:
Good to see that we're raising more or less the same questions – I didn't read your review comments before my review in https://github.com/netzstrategen/drupal-twig_fractal/pull/1#discussion_r134539185 👍